src/page-response-impact/rtStatistics.tsx

```diff
@@ -0,0 +1,42 @@
+import { RtRecord } from "../infection-model/rt";
+
+export function numFacilitiesWithRtLessThan1(
+  facilitiesRtRecords: RtRecord[][],
+) {
+  const latestRtValues = facilitiesRtRecords.map((rtRecords) => {
+    // Note: This assumes the first in the array is the latest because of the
+    // sorting on line 104 in infection-model/rt.tsx
+    return rtRecords[0].value;
+  });
+  return latestRtValues.filter((v) => v < 1).length;
+}
+
+export function averageRtReductionAcrossFacilities(
+  facilitiesRtRecords: RtRecord[][],
+) {
+  const numFacilities = facilitiesRtRecords.length;
+  const rtDifferences: number[] = facilitiesRtRecords.map(
+    (rtRecords: RtRecord[]) => {
+      const latestRtValue = rtRecords[0].value;
+      const oldestRtValue = rtRecords[rtRecords.length - 1].value;
+      return oldestRtValue - latestRtValue;
+    },
+  );
+  const sumDifferences = rtDifferences.reduce(
+    (sum, rtDifference) => sum + rtDifference,
+    0,
+  );
+  return sumDifferences / numFacilities;
```

Comment:
this is fine but calling `lodash.mean` or `d3.mean` on `rtDifferences` would be more concise. at the very least we should use `rtDifferences.length` rather than `numFacilities` in case there is a future change such that they get out of sync (like, we drop invalid values from the list of differences or something like that ... it's not completely far-fetched)